tests/e2e/tests/content-manager/editview.spec.ts

```diff
@@ -129,7 +129,6 @@ test.describe('Edit View', () => {
       );
       await expect(page.getByRole('button', { name: 'Save' })).toBeDisabled();
       await expect(page.getByRole('button', { name: 'Publish' })).toBeDisabled();
-      await expect(page.getByRole('button', { name: 'More document actions' })).toBeDisabled();
```

Comment:
Why removing the assertion instead of checking if the button is enabled and just add a comment to explain why this one is enabled

Reply:
imo I don't see the point of checking that it's enabled, I think it makes more sense to keep the set of expectations consistent

but I'm still wondering should the releases button be enabled when we're on the published tab?

Reply:
yeah you can unpublish stuff with release

Reply:
@simotae14 I see thanks. To me removing these expectations makes sense then. Could you approve if you're okay with the change?